packages/yourloops/lib/data/api.ts

```diff
@@ -319,18 +314,9 @@ export async function getMessageThread(session: Session, messageId: string): Pro
   });
 
   if (response.ok) {
-    const out = (await response.json()) as MessagesThread | undefined;
-    const messages: MessageNote[] = out?.messages ?? [];
-    if (!Array.isArray(messages)) {
-      log.error("Expected an array of messages", { messages });
-      Promise.reject(new Error("Invalid response"));
-    }
+    const messages = (await response.json()) as MessageNote[];
```

Comment:
Can we trust this new service to return an array ?
We don't want to crash if it's not the case.

Reply:
:+1:

Reply:
Updated